test/new-e2e/tests/installer/all_packages_test.go

```diff
@@ -61,8 +61,8 @@ var (
 )
 
 var packagesConfig = []testPackageConfig{
-	{name: "datadog-installer", defaultVersion: fmt.Sprintf("pipeline-%v", os.Getenv("CI_PIPELINE_ID")), registry: "669783387624.dkr.ecr.us-east-1.amazonaws.com", auth: "ecr"},
-	{name: "datadog-agent", defaultVersion: fmt.Sprintf("pipeline-%v", os.Getenv("CI_PIPELINE_ID")), registry: "669783387624.dkr.ecr.us-east-1.amazonaws.com", auth: "ecr"},
+	{name: "datadog-installer", defaultVersion: fmt.Sprintf("pipeline-%v", os.Getenv("E2E_PIPELINE_ID")), registry: "669783387624.dkr.ecr.us-east-1.amazonaws.com", auth: "ecr"},
```

Comment:
Is this new variable guaranteed? Wdyt, if we fallback to the `CI_PIPELINE_ID`?
As this will be processed on package load

Reply:
I think this variable can be considered guaranteed in the CI, as long as your job extends `.new-e2e_template` the variable will exist. And if you do not extend that job it will probably not work because other variables will be missing.